modules/infra/main.tf

```diff
@@ -11,6 +11,18 @@ locals {
   }
 }
 
+module "cost_usage_report" {
+  count        = var.domino_cur.provision_resources ? 1 : 0
+  source       = "./submodules/cost-usage-report"
+  deploy_id    = var.deploy_id
+  network_info = module.network.info
+  kms_info     = local.kms_info
+  region       = var.region
+  providers = {
+    aws.domino_cur_region = aws.us-east-1
```

Comment:
I would rename this alias `aws.domino_cur_region` -> `aws.us-east-1` given not all resources are using it.